// @shoelace-style/shoelace/cdn/react/dropdown/index.js
import {
  SlDropdown
} from "../../chunks/chunk.2X7HEV5P.js";
import "../../chunks/chunk.WMFHBY7P.js";
import "../../chunks/chunk.YCHBWCKL.js";
import "../../chunks/chunk.YUZUQX7V.js";
import "../../chunks/chunk.6LJMKLWZ.js";
import "../../chunks/chunk.52GJFLW5.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.OZPGMRHW.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/dropdown/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-dropdown";
SlDropdown.define("sl-dropdown");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlDropdown,
  react: React,
  events: {
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide"
  },
  displayName: "SlDropdown"
});
var dropdown_default = reactWrapper;
export {
  dropdown_default as default
};
