// @shoelace-style/shoelace/dist/react/input/index.js
import {
  SlInput
} from "../../chunks/chunk.QRWOGTXF.js";
import "../../chunks/chunk.5NGT6QDY.js";
import "../../chunks/chunk.SI4ACBFK.js";
import "../../chunks/chunk.GI7VDIWX.js";
import "../../chunks/chunk.UEQZEZFU.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/input/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-input";
SlInput.define("sl-input");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlInput,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlClear: "sl-clear",
    onSlFocus: "sl-focus",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlInput"
});
var input_default = reactWrapper;
export {
  input_default as default
};
