// @shoelace-style/shoelace/dist/react/tooltip/index.js
import {
  SlTooltip
} from "../../chunks/chunk.OFUC6WJC.js";
import "../../chunks/chunk.NY74SEUT.js";
import "../../chunks/chunk.V3BMAKGV.js";
import "../../chunks/chunk.NJGFDSYD.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/tooltip/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-tooltip";
SlTooltip.define("sl-tooltip");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlTooltip,
  react: React,
  events: {
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide"
  },
  displayName: "SlTooltip"
});
var tooltip_default = reactWrapper;
export {
  tooltip_default as default
};
