// @shoelace-style/shoelace/cdn/react/input/index.js
import {
  SlInput
} from "../../chunks/chunk.AJ5LIZTV.js";
import "../../chunks/chunk.2SZPKEN5.js";
import "../../chunks/chunk.XQ4KBI5G.js";
import "../../chunks/chunk.HPQ3PGLN.js";
import "../../chunks/chunk.RIBKWZXP.js";
import "../../chunks/chunk.3HJZQ2YA.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.PNPJ7KDG.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.NX77NOIW.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.24R5WE5A.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/input/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-input";
SlInput.define("sl-input");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlInput,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlClear: "sl-clear",
    onSlFocus: "sl-focus",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlInput"
});
var input_default = reactWrapper;
export {
  input_default as default
};
