// @shoelace-style/shoelace/cdn/react/copy-button/index.js
import {
  SlCopyButton
} from "../../chunks/chunk.PR4JUEF5.js";
import "../../chunks/chunk.ZRNKDY5G.js";
import "../../chunks/chunk.GL3UMKIL.js";
import "../../chunks/chunk.EFUIYJLS.js";
import "../../chunks/chunk.YUZUQX7V.js";
import "../../chunks/chunk.6LJMKLWZ.js";
import "../../chunks/chunk.52GJFLW5.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.OZPGMRHW.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.NX77NOIW.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.24R5WE5A.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/copy-button/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-copy-button";
SlCopyButton.define("sl-copy-button");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlCopyButton,
  react: React,
  events: {
    onSlCopy: "sl-copy",
    onSlError: "sl-error"
  },
  displayName: "SlCopyButton"
});
var copy_button_default = reactWrapper;
export {
  copy_button_default as default
};
