// @shoelace-style/shoelace/dist/react/copy-button/index.js
import {
  SlCopyButton
} from "../../chunks/chunk.JYBPFYZI.js";
import "../../chunks/chunk.OFUC6WJC.js";
import "../../chunks/chunk.NY74SEUT.js";
import "../../chunks/chunk.PH5IIYTE.js";
import "../../chunks/chunk.V3BMAKGV.js";
import "../../chunks/chunk.NJGFDSYD.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/copy-button/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-copy-button";
SlCopyButton.define("sl-copy-button");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlCopyButton,
  react: React,
  events: {
    onSlCopy: "sl-copy",
    onSlError: "sl-error"
  },
  displayName: "SlCopyButton"
});
var copy_button_default = reactWrapper;
export {
  copy_button_default as default
};
