// @shoelace-style/shoelace/dist/react/tree-item/index.js
import {
  SlTreeItem
} from "../../chunks/chunk.IUTSML4E.js";
import "../../chunks/chunk.BXQVOJJ6.js";
import "../../chunks/chunk.5LKA6J7H.js";
import "../../chunks/chunk.YHQNI3TZ.js";
import "../../chunks/chunk.GI7VDIWX.js";
import "../../chunks/chunk.J2B3BBYS.js";
import "../../chunks/chunk.CAHT2E74.js";
import "../../chunks/chunk.UEQZEZFU.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/tree-item/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-tree-item";
SlTreeItem.define("sl-tree-item");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlTreeItem,
  react: React,
  events: {
    onSlExpand: "sl-expand",
    onSlAfterExpand: "sl-after-expand",
    onSlCollapse: "sl-collapse",
    onSlAfterCollapse: "sl-after-collapse",
    onSlLazyChange: "sl-lazy-change",
    onSlLazyLoad: "sl-lazy-load"
  },
  displayName: "SlTreeItem"
});
var tree_item_default = reactWrapper;
export {
  tree_item_default as default
};
