// @shoelace-style/shoelace/dist/react/color-picker/index.js
import {
  SlColorPicker
} from "../../chunks/chunk.IOWD7PKG.js";
import "../../chunks/chunk.IWC2ICZZ.js";
import "../../chunks/chunk.IOFPPNUO.js";
import "../../chunks/chunk.A4SOQOK5.js";
import "../../chunks/chunk.QRWOGTXF.js";
import "../../chunks/chunk.5NGT6QDY.js";
import "../../chunks/chunk.SI4ACBFK.js";
import "../../chunks/chunk.7IB76QFQ.js";
import "../../chunks/chunk.O6IUBC6G.js";
import "../../chunks/chunk.YCHBWCKL.js";
import "../../chunks/chunk.GI7VDIWX.js";
import "../../chunks/chunk.BYIKSKAM.js";
import "../../chunks/chunk.V3BMAKGV.js";
import "../../chunks/chunk.NJGFDSYD.js";
import "../../chunks/chunk.HF7GESMZ.js";
import "../../chunks/chunk.77MNWP7M.js";
import "../../chunks/chunk.J2B3BBYS.js";
import "../../chunks/chunk.CAHT2E74.js";
import "../../chunks/chunk.UEQZEZFU.js";
import "../../chunks/chunk.MGOBPGE5.js";
import "../../chunks/chunk.XPS2TE3G.js";
import "../../chunks/chunk.WISH4DLW.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/color-picker/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-color-picker";
SlColorPicker.define("sl-color-picker");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlColorPicker,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlFocus: "sl-focus",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlColorPicker"
});
var color_picker_default = reactWrapper;
export {
  color_picker_default as default
};
