// @shoelace-style/shoelace/cdn/react/button/index.js
import {
  SlButton
} from "../../chunks/chunk.EDCUTCXJ.js";
import "../../chunks/chunk.QDF3TSOY.js";
import "../../chunks/chunk.3GITBIC4.js";
import "../../chunks/chunk.3HJZQ2YA.js";
import "../../chunks/chunk.BXWBYEBW.js";
import "../../chunks/chunk.MKSGYDJ7.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.PNPJ7KDG.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.NX77NOIW.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.24R5WE5A.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/button/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-button";
SlButton.define("sl-button");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlButton,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlFocus: "sl-focus",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlButton"
});
var button_default = reactWrapper;
export {
  button_default as default
};
