// @shoelace-style/shoelace/dist/react/checkbox/index.js
import {
  SlCheckbox
} from "../../chunks/chunk.5LKA6J7H.js";
import "../../chunks/chunk.YHQNI3TZ.js";
import "../../chunks/chunk.GI7VDIWX.js";
import "../../chunks/chunk.UEQZEZFU.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/checkbox/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-checkbox";
SlCheckbox.define("sl-checkbox");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlCheckbox,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlFocus: "sl-focus",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlCheckbox"
});
var checkbox_default = reactWrapper;
export {
  checkbox_default as default
};
