// @shoelace-style/shoelace/dist/react/alert/index.js
import {
  SlAlert
} from "../../chunks/chunk.LEGAY6NB.js";
import "../../chunks/chunk.F6JSOJOM.js";
import "../../chunks/chunk.7BLP64MK.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.4GRKCE6W.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/alert/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-alert";
SlAlert.define("sl-alert");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlAlert,
  react: React,
  events: {
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide"
  },
  displayName: "SlAlert"
});
var alert_default = reactWrapper;
export {
  alert_default as default
};
