// @shoelace-style/shoelace/cdn/react/range/index.js
import {
  SlRange
} from "../../chunks/chunk.4OIXEKO3.js";
import "../../chunks/chunk.DAPUP5RK.js";
import "../../chunks/chunk.XQ4KBI5G.js";
import "../../chunks/chunk.HPQ3PGLN.js";
import "../../chunks/chunk.RIBKWZXP.js";
import "../../chunks/chunk.3HJZQ2YA.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.PNPJ7KDG.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/range/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-range";
SlRange.define("sl-range");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlRange,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlFocus: "sl-focus",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlRange"
});
var range_default = reactWrapper;
export {
  range_default as default
};
