// @shoelace-style/shoelace/cdn/react/switch/index.js
import {
  SlSwitch
} from "../../chunks/chunk.G3HDDXER.js";
import "../../chunks/chunk.FSFPD2YS.js";
import "../../chunks/chunk.HPQ3PGLN.js";
import "../../chunks/chunk.RIBKWZXP.js";
import "../../chunks/chunk.3HJZQ2YA.js";
import "../../chunks/chunk.PNPJ7KDG.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/switch/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-switch";
SlSwitch.define("sl-switch");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlSwitch,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlInput: "sl-input",
    onSlFocus: "sl-focus",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlSwitch"
});
var switch_default = reactWrapper;
export {
  switch_default as default
};
