// @shoelace-style/shoelace/cdn/react/select/index.js
import {
  SlSelect
} from "../../chunks/chunk.FBUDGD47.js";
import "../../chunks/chunk.EOM4EELX.js";
import "../../chunks/chunk.4N33ARUV.js";
import "../../chunks/chunk.25MM5II7.js";
import "../../chunks/chunk.4AL4ELCP.js";
import "../../chunks/chunk.RK73WSZS.js";
import "../../chunks/chunk.XQ4KBI5G.js";
import "../../chunks/chunk.RIBKWZXP.js";
import "../../chunks/chunk.YUZUQX7V.js";
import "../../chunks/chunk.6LJMKLWZ.js";
import "../../chunks/chunk.3HJZQ2YA.js";
import "../../chunks/chunk.KBEINLRV.js";
import "../../chunks/chunk.FDFOFJDR.js";
import "../../chunks/chunk.52GJFLW5.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.OZPGMRHW.js";
import "../../chunks/chunk.MKSGYDJ7.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.PNPJ7KDG.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.NX77NOIW.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.24R5WE5A.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/select/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-select";
SlSelect.define("sl-select");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlSelect,
  react: React,
  events: {
    onSlChange: "sl-change",
    onSlClear: "sl-clear",
    onSlInput: "sl-input",
    onSlFocus: "sl-focus",
    onSlBlur: "sl-blur",
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlSelect"
});
var select_default = reactWrapper;
export {
  select_default as default
};
