// @shoelace-style/shoelace/cdn/react/animated-image/index.js
import {
  SlAnimatedImage
} from "../../chunks/chunk.JLNRTLPL.js";
import "../../chunks/chunk.NX77NOIW.js";
import "../../chunks/chunk.TDLFM7SW.js";
import "../../chunks/chunk.24R5WE5A.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.JS5TXHUY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/animated-image/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-animated-image";
SlAnimatedImage.define("sl-animated-image");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlAnimatedImage,
  react: React,
  events: {
    onSlLoad: "sl-load",
    onSlError: "sl-error"
  },
  displayName: "SlAnimatedImage"
});
var animated_image_default = reactWrapper;
export {
  animated_image_default as default
};
