// @shoelace-style/shoelace/dist/react/dropdown/index.js
import {
  SlDropdown
} from "../../chunks/chunk.7IB76QFQ.js";
import "../../chunks/chunk.O6IUBC6G.js";
import "../../chunks/chunk.YCHBWCKL.js";
import "../../chunks/chunk.V3BMAKGV.js";
import "../../chunks/chunk.NJGFDSYD.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/dropdown/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-dropdown";
SlDropdown.define("sl-dropdown");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlDropdown,
  react: React,
  events: {
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide"
  },
  displayName: "SlDropdown"
});
var dropdown_default = reactWrapper;
export {
  dropdown_default as default
};
