// @shoelace-style/shoelace/cdn/react/tooltip/index.js
import {
  SlTooltip
} from "../../chunks/chunk.ZRNKDY5G.js";
import "../../chunks/chunk.GL3UMKIL.js";
import "../../chunks/chunk.YUZUQX7V.js";
import "../../chunks/chunk.6LJMKLWZ.js";
import "../../chunks/chunk.52GJFLW5.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.OZPGMRHW.js";
import "../../chunks/chunk.BQBM4GHU.js";
import "../../chunks/chunk.JRE37J4J.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/tooltip/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-tooltip";
SlTooltip.define("sl-tooltip");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlTooltip,
  react: React,
  events: {
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide"
  },
  displayName: "SlTooltip"
});
var tooltip_default = reactWrapper;
export {
  tooltip_default as default
};
