// @shoelace-style/shoelace/dist/react/select/index.js
import {
  SlSelect
} from "../../chunks/chunk.QIAOCRSS.js";
import "../../chunks/chunk.MTM3FR3I.js";
import "../../chunks/chunk.2FVP4SGD.js";
import "../../chunks/chunk.YZQJI64O.js";
import "../../chunks/chunk.RK73WSZS.js";
import "../../chunks/chunk.SI4ACBFK.js";
import "../../chunks/chunk.GI7VDIWX.js";
import "../../chunks/chunk.V3BMAKGV.js";
import "../../chunks/chunk.NJGFDSYD.js";
import "../../chunks/chunk.UEQZEZFU.js";
import "../../chunks/chunk.F6JSOJOM.js";
import "../../chunks/chunk.7BLP64MK.js";
import "../../chunks/chunk.H437TPPF.js";
import "../../chunks/chunk.B4BZKR24.js";
import "../../chunks/chunk.KRRLOROJ.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.OSHRQMHR.js";
import "../../chunks/chunk.LMJPQPQT.js";
import "../../chunks/chunk.3WAW4E2K.js";
import "../../chunks/chunk.P7ZG6EMR.js";
import "../../chunks/chunk.VRTJZYIC.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.3Y6SB6QS.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/select/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-select";
SlSelect.define("sl-select");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlSelect,
  react: React,
  events: {
    onSlChange: "sl-change",
    onSlClear: "sl-clear",
    onSlInput: "sl-input",
    onSlFocus: "sl-focus",
    onSlBlur: "sl-blur",
    onSlShow: "sl-show",
    onSlAfterShow: "sl-after-show",
    onSlHide: "sl-hide",
    onSlAfterHide: "sl-after-hide",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlSelect"
});
var select_default = reactWrapper;
export {
  select_default as default
};
