// @shoelace-style/shoelace/dist/react/range/index.js
import {
  SlRange
} from "../../chunks/chunk.W2QSM3ZN.js";
import "../../chunks/chunk.WB6EPZFH.js";
import "../../chunks/chunk.SI4ACBFK.js";
import "../../chunks/chunk.GI7VDIWX.js";
import "../../chunks/chunk.UEQZEZFU.js";
import "../../chunks/chunk.2A3352EO.js";
import "../../chunks/chunk.MAS2SHYD.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.Q6ASPMKT.js";
import "../../chunks/chunk.WQPSQJ5O.js";
import "../../chunks/chunk.TUVJKY7S.js";
import "../../chunks/chunk.MAD5PUM2.js";

// src/react/range/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-range";
SlRange.define("sl-range");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlRange,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlChange: "sl-change",
    onSlFocus: "sl-focus",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlRange"
});
var range_default = reactWrapper;
export {
  range_default as default
};
