// @shoelace-style/shoelace/cdn/react/radio-group/index.js
import {
  SlRadioGroup
} from "../../chunks/chunk.CLRQKJDN.js";
import "../../chunks/chunk.4FM4N6Z2.js";
import "../../chunks/chunk.XQ4KBI5G.js";
import "../../chunks/chunk.3HJZQ2YA.js";
import "../../chunks/chunk.OSYMSUY5.js";
import "../../chunks/chunk.HQAAUCQS.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/radio-group/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-radio-group";
SlRadioGroup.define("sl-radio-group");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlRadioGroup,
  react: React,
  events: {
    onSlChange: "sl-change",
    onSlInput: "sl-input",
    onSlInvalid: "sl-invalid"
  },
  displayName: "SlRadioGroup"
});
var radio_group_default = reactWrapper;
export {
  radio_group_default as default
};
