// @shoelace-style/shoelace/cdn/react/radio-button/index.js
import {
  SlRadioButton
} from "../../chunks/chunk.MJA6RIPI.js";
import "../../chunks/chunk.UT2XRHAI.js";
import "../../chunks/chunk.BXWBYEBW.js";
import "../../chunks/chunk.MKSGYDJ7.js";
import "../../chunks/chunk.PNPJ7KDG.js";
import "../../chunks/chunk.NYIIDP5N.js";
import "../../chunks/chunk.DOYC4G7X.js";
import "../../chunks/chunk.JJNPZ52S.js";
import "../../chunks/chunk.XAOA43RZ.js";
import "../../chunks/chunk.SVM3EOMY.js";
import "../../chunks/chunk.M2U2QT2K.js";
import "../../chunks/chunk.27ILGUWR.js";
import "../../chunks/chunk.YZETUBD6.js";

// src/react/radio-button/index.ts
import * as React from "react";
import { createComponent } from "@lit/react";
import "@lit/react";
var tagName = "sl-radio-button";
SlRadioButton.define("sl-radio-button");
var reactWrapper = createComponent({
  tagName,
  elementClass: SlRadioButton,
  react: React,
  events: {
    onSlBlur: "sl-blur",
    onSlFocus: "sl-focus"
  },
  displayName: "SlRadioButton"
});
var radio_button_default = reactWrapper;
export {
  radio_button_default as default
};
